openapi/components/requestBodies/Webhooks/InvoiceTaxCalculationFailed.yaml

```
@@ -0,0 +1,26 @@
+content:
+  application/json:
+    schema:
+      properties:
+        invoiceId:
+          description: The ID of the invoice.
+          type: string
+        reason:
+          description: Error message explaining tax calculation failure.
+          type: string
+        eventType:
+          $ref: ../../schemas/GlobalWebhookEventType.yaml
+        _embedded:
+          type: object
+          description: Invoice object.
+          properties:
+            invoice:
+              $ref: ../../schemas/Common/CommonInvoice.yaml
```

```suggestion
              $ref: ../../schemas/Invoices/Invoice.yaml
```
I think it should use Invoice schema, not common